packages/editor-ui/src/views/NodeView.vue

```diff
@@ -307,9 +312,15 @@ export default mixins(
 			}
 		},
 		computed: {
+			...mapGetters('users', [
+				'currentUser',
+			]),
 			...mapGetters('ui', [
 				'sidebarMenuCollapsed',
 			]),
+			...mapGetters('settings', [
+				'isOnboardingCallPromptFeatureEnabled',
+			]),
```

Comment:
I have stopped liking `mapGetters` because it does not give types.. instead manually adding computed functions to set types manually instead..  (just a note, you don't have to change this)

Reply:
Ok, agree for the future uses.